robots/flakefinder/main.go

```diff
@@ -0,0 +1,143 @@
+/*
+ * This file is part of the KubeVirt project
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ * Copyright 2019 Red Hat, Inc.
+ *
+ */
+
+package main
+
+import (
+	"context"
+	"flag"
+	"fmt"
+	"log"
+	"net/url"
+	"strings"
+	"time"
+
+	"cloud.google.com/go/storage"
+	"k8s.io/test-infra/prow/config/secret"
+	"k8s.io/test-infra/prow/flagutil"
+	"k8s.io/test-infra/prow/github"
+)
+
+func flagOptions() options {
+	o := options{
+		endpoint: flagutil.NewStrings("https://api.github.com"),
+	}
+	flag.IntVar(&o.ceiling, "ceiling", 100, "Maximum number of issues to modify, 0 for infinite")
+	flag.DurationVar(&o.merged, "merged", 24*7*time.Hour, "Filter to issues merged in the time window")
+	flag.Var(&o.endpoint, "endpoint", "GitHub's API endpoint")
+	flag.StringVar(&o.token, "token", "", "Path to github token")
+	flag.StringVar(&o.graphqlEndpoint, "graphql-endpoint", github.DefaultGraphQLEndpoint, "GitHub's GraphQL API Endpoint")
+	flag.Parse()
+	return o
+}
+
+type options struct {
+	ceiling         int
+	endpoint        flagutil.Strings
+	token           string
+	graphqlEndpoint string
+	merged          time.Duration
+}
+
+type client interface {
```

Comment:
could you format that interface that methods have the same indents number?

Reply:
Ah interesting, looks like a tabs vs. spaces issue. Will correct this.